src/documents/index.md

```diff
@@ -1,6 +1,7 @@
 ---
 title: Infusion Documentation
 layout: default
+category: Infusion
 ---
 
-Welcome to the Infusion Documentation.
+This is documentation related to using, configuring, and customizing Infusion. Use the table of contents to navigate the content.
```

Comment:
Perhaps all of these "section introduction" pages could say something more like "This section contains documentation related to..." instead of "This is..."